### PR TITLE
ICP-1124 - Sylvania SMART BR30 RGBW ' s color ticker moving randomly

### PR DESCRIPTION
**Pull request by juano2310**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 1]
```diff
@@ -103,12 +103,12 @@ def parse(String description) {
 
         if (zigbeeMap?.clusterInt == COLOR_CONTROL_CLUSTER) {
             if(zigbeeMap.attrInt == ATTRIBUTE_HUE){  //Hue Attribute
-                def hueValue = Math.round(zigbee.convertHexToInt(zigbeeMap.value) / 0xfe * 100)
-                sendEvent(name: "hue", value: hueValue, descriptionText: "Color has changed")
+                state.hueValue = Math.round(zigbee.convertHexToInt(zigbeeMap.value) / 0xfe * 100)
+                runIn(5, updateColor, [overwrite: true])
```
Comment (marstorp) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy:107]: Would be nice with a comment on why runIn() is used and why the delay is set to 5 seconds, for maintainability purpose

**Review by marstorp — APPROVED** [review]
Nitpicking but for maintainability comments are always nice to have for the next person when it's not obvious why runIn is used.

**Comment by juano2310** [comment]
@workingmonk do you mind merging this?

**Comment by juano2310** [comment]
@tpmanley what do you think about this solution to prevent the color picker in SC from jumping when getting saturation and hue in separated events? It works but wondering if this can affect something else.

**Comment by tpmanley** [comment]
I don't have any objections

**Pull request merged**